app/utils/appstatus/fast_storage.go

```diff
@@ -0,0 +1,40 @@
+package appstatus
+
+import (
+	"fmt"
+	"path"
+
+	"github.com/okex/exchain/libs/cosmos-sdk/client/flags"
+	sdk "github.com/okex/exchain/libs/cosmos-sdk/types"
+	"github.com/okex/exchain/libs/cosmos-sdk/x/auth"
+	"github.com/okex/exchain/libs/iavl"
+	dbm "github.com/okex/exchain/libs/tm-db"
+	"github.com/okex/exchain/x/evm"
+	"github.com/spf13/viper"
+)
+
+func IsFastStorageStrategy() bool {
+	home := viper.GetString(flags.FlagHome)
+	dataDir := path.Join(home, "data")
+	db, err := sdk.NewLevelDB("application", dataDir)
```

Comment:
can use const val

Reply:
have fixed. use const val `dbFolder` and `applicationDB` instead of hard code